static/app/views/insights/uptime/views/overview.spec.tsx

```diff
@@ -70,15 +70,23 @@ describe('Uptime Overview', function () {
 
   it('renders', async function () {
     const {organization, router} = initializeOrg({
-      organization: {features: ['insights-initial-modules', 'insights-uptime']},
+      organization: {
+        features: [
+          'insights-initial-modules',
+          'insights-entry-points',
+          'insights-uptime',
+        ],
+      },
     });
     OrganizationStore.onUpdate(organization);
 
     render(<UptimeOverview />, {organization, router});
-    expect(
-      await screen.findByRole('heading', {name: 'Uptime Monitors'})
-    ).toBeInTheDocument();
 
-    expect(screen.getByRole('heading', {name: 'Test Monitor'})).toBeInTheDocument();
```

Comment:
@evanpurkhiser lmk if this PR breaks something here

I was a bit confused about this test case, we expect the `heading` to be `Uptime Monitors` and then we expect the `heading` to be `Test Monitor` directly below?